types/rdfjs__data-model/rdfjs__data-model-tests.ts

```diff
@@ -0,0 +1,48 @@
+import factory from "@rdfjs/data-model";
+import * as RDF from "@rdfjs/types";
+
+const myQuad = factory.quad<RDF.Quad>(
```

Comment:
Ok, so please disregard my earlier comments. I think we're good and might improve later if anyone has problem with the generic types.

For the sake of tests I would only propose that you remove the explicit type param where unnecessary to actually verify its correct default. Here and below

```suggestion
const myQuad: RDF.Quad = factory.quad(
```

Reply:
Thanks for all the reviews! I believe that I have now incorporated all relevant ones, let me know if not :)